pyap/source_US/data.py

```diff
@@ -140,6 +140,8 @@ def str_list_to_upper_lower_regex(str_list: List[str]) -> str:
 
 highway_re = r"""(?:[Hh][Ii][Gg][Hh][Ww][Aa][Yy]\s+\d{1,4})"""
 
+avenue_re = r"""(?:[NSWE]\ [Aa][Vv][Ee](?:\.|[Nn][Uu][Ee])?\ \d{1,2})"""
```

Comment:
maybe you could use `post_direction_re` instead of `[NSWE]`?
Also maybe this should be named something like `numbered_avenue_re` instead.